### ISSUE TITLE
Feature Request : Incorporate slideshows and let user pick image

### ISSUE THREAD

**Issue by Arcadia197** [pratap-panabaka/gse-lockscreen-extension#2]
I love the work that you did here! I have two feature requests that I would like to propose:

1. I have a slideshow enabled (created with shotwell) to cycle through backgrounds. When choosing the standard wallpaper (from org.gnome.desktop.background.picture-uri) it should point to that, however I see the ubuntu preset-wallpaper. Does your implementation not support slideshows yet? Would love to see that!
2. How about you incorporate a Button where the user can either pick a file or write in a path to choose from in order to set their own wallpapers in the selection menu? Think that would easily increase the flexibility as well.

I guess with those two features people that would like to highly customize the wallpapers themself could greatly use your extension, to decrease the blur effect on the screensaver (and therefore enjoy the images in full quality when it is locked)

**Comment by PRATAP-KUMAR** [comment]
Hi @Arcadia197 I did try yesterday for the slide show and crashed system with the while loop. Thats Ok.
So now to make it a corner, slide show you want to achieve with "org.gnome.desktop.background.picture-uri" this key choosing the xml file rite? Means you want the same behaviour of desktop background changes with the specified time on lockscreen too rite?

**Comment by Arcadia197** [comment]
Exactly! That's what ultimately would be the most convenient, I think - It would also give the same behaviour as the overall gnome desktop background when the screen is not locked, but with all your extra settings on top!

**Comment by PRATAP-KUMAR** [comment]
Ok, Clear. I will try and get back to you.

**Comment by PRATAP-KUMAR** [comment]
@Arcadia197 Sorry I am not getting free time to work on this. It will take much time. I will ping you when ever I get a chance to work on it.

Thanks

**Comment by pratap-panabaka** [comment]
Hi @Arcadia197 Your slideshow should work with this extensions version 9 update.
Also for picking up the image, there is a work around now which is possible with this version 9.

**Comment by pratap-panabaka** [comment]
@Arcadia197 for slideshow it is not possible if you are using `.xml` file but when you use [variety app](https://github.com/varietywalls/variety) the slideshow works.
